core/src/main/scala/kafka/coordinator/PidMetadata.scala

```diff
@@ -19,10 +19,17 @@ package kafka.coordinator
 import kafka.utils.nonthreadsafe
 
 @nonthreadsafe
-private[coordinator] class PidMetadata(val pid: Long) {
+private[coordinator] class PidMetadata(val pid: Long,
```

Comment:
I do not have a strong opinion here, so I'm fine with separating them. Could you propose the new package names as I'm not good at this? :P